build-custom-image/workflow.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: batch.tensorstack.dev/v1beta1
+kind: WorkflowTemplate
+metadata:
+  labels:
+    workflowtemplate/tags: image-build
+  name: build-custom-image
+spec:
+  params:
+    - default: Dockerfile
+      description: The relative path of the Dockerfile in the build context.
+      name: dockerfile
+    - default: 'registry.tensorstack.cn/t9kmirror/kaniko-project-executor:latest'
+      description: Kaniko builder image.
+      name: builderImage
+    - default: ''
+      description: Destination image name.
+      name: dstImage
+    - default: .
+      description: Build context used by Kaniko.
+      name: contextPath
+  results:
+    - description: Digest of the image just built.
+      name: imageDigest
+    - description: URL of the image just built.
+      name: imageURL
+  seqPod:
+    steps:
+      - args:
+          - '--context=$(workspaces.source.path)/$(params.contextPath)'
+          - '--destination=$(params.dstImage)'
+          - '--dockerfile=$(workspaces.source.path)/$(params.contextPath)/$(params.dockerfile)'
+          - '--digest-file=$(results.imageDigest.path)'
+        command:
+          - /kaniko/executor
+        env:
+          - name: DOCKER_CONFIG
+            value: $(workspaces.dockerConfig.path)
+        image: $(params.builderImage)
+        name: builder
+        resources:
+          limits:
```

Comment:
Set `resource.request`, because if `requests` is not provided, it will take value from `limits`, which is pretty high for memory.